.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
A thousand bonus points for having this in here and therefore not pushing 1500 `node_module` files with your commit.